homeassistant/components/meraki/device_tracker.py

```diff
@@ -64,7 +65,7 @@ async def post(self, request):
         if data["secret"] != self.secret:
             _LOGGER.error("Invalid Secret received from Meraki")
             return self.json_message("Invalid secret", HTTP_UNPROCESSABLE_ENTITY)
-        if data["version"] != VERSION:
+        if int(Decimal(data["version"])) != int(Decimal(VERSION)):
```

Comment:
I suggest using AwesomeVersion for version parsing. That's our go to tool for that and available as part of core requirements.

https://github.com/ludeeus/awesomeversion

Reply:
Another option is [packaging](https://packaging.pypa.io/en/latest/version.html)

Reply:
Thanks, Updated code to use AwesomeVersion